src/components/landing/pages/history/common/entry-menu/delete-remove-note-item.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Fragment, useState } from 'react'
+import { Dropdown } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { ForkAwesomeIcon, IconName } from '../../../../../common/fork-awesome/fork-awesome-icon'
+import { DeletionModal } from '../../../../../common/modals/deletion-modal'
+import { ShowIf } from '../../../../../common/show-if/show-if'
+
+export interface DeleteRemoveNoteItemProps {
+  onConfirm: () => void
+  noteTitle: string
+  variant: 'removeHistoryItem' | 'deleteNote'
+}
+
+export const DeleteRemoveNoteItem: React.FC<DeleteRemoveNoteItemProps> = ({ variant, onConfirm, noteTitle }) => {
```

Comment:
you maybe want to look at the last thread in my earlier review

Reply:
Sry but I think it's better to split components that are meant for different semantic tasks.

Reply:
how about this then, have two components that only use on common component with all the necessary props. That way there are two components, but since they use exactly the same logic the base is the same and we minimize repeating code.

Reply:
The react way to do this is composition. It's totally legit to create an abstract component with many props and then a specific component that just wraps this abstract component with values.

Reply:
Isn't that what I described?

Reply:
Yes. Github hasn't shown your comment when I wrote mine.